.github/workflows/int-and-connected-test-run.yml

```diff
@@ -7,6 +7,9 @@ env:
   GOOGLE_SA_CERT: 'jade-dev-account.pem'
   # This will make test logs not use JSON
   TDR_LOG_APPENDER: 'Console-Standard'
+  # The default Azure credentials to use to authenticate
+  AZURE_CREDENTIALS_APPLICATIONID: 22cb243c-f1a5-43d8-8f12-6566bcce6542
```

Comment:
So are these public IDs that are part of the azure interface? Or are these IDs that we generate somehow?

Reply:
These are IDs that are generated for us by Azure

Reply:
I didn't think these needed to be secrets (but could certainly move them there if needed)

Reply:
Well, they are public now, as they've been committed to a public repo. If you were running `git secrets` I think they would've been flagged, and you would've had to add an exception for them. 

As long as they're unusable without another secret, it seems ok to have them in the source. Is it kind of like committing a username?

Reply:
Right exactly

Reply:
FWIW, confirmed with msft folks too